spring-ai-alibaba-graph-core/src/main/java/com/alibaba/cloud/ai/graph/GraphRunnerContext.java

```diff
@@ -93,20 +93,24 @@ private void initializeFromResume(OverAllState initialState, RunnableConfig conf
 
 		var saver = compiledGraph.compileConfig.checkpointSaver()
 				.orElseThrow(() -> new IllegalStateException("Resume request without a configured checkpoint saver!"));
-		var checkpoint = saver.get(config)
+        
+		var lookupConfig = config.metadata("SOURCE_THREAD_ID")
```

Comment:
What's `SOURCE_THREAD_ID`? What's the difference between `SOURCE_THREAD_ID` and the normal `threadId`? 

Could you please help to explain?